src/libs/actions/IOU.js

```diff
@@ -111,8 +111,32 @@ function createIOUSplit(params) {
         });
 }
 
+/**
+ * Settles an IOU Report. As we can guarantee that the settled report was previously active,
+ * we should update the chatReport data too when calling `fetchIOUReportByID`.
+ */
+function settleIOUReport({
+    chatReportID, reportID, paymentMethodType,
+}) {
+    Onyx.merge(ONYXKEYS.IOU, {loading: true, error: false});
+    API.PayIOU({
+        reportID,
+        paymentMethodType,
+    })
+        .then((data) => {
+            if (data.jsonCode !== 200) {
+                console.error(data.message);
+            }
+        })
+        .then(fetchChatReportsByIDs([chatReportID]))
+        .then(fetchIOUReportByID(reportID, chatReportID, true))
```

Comment:
The functions are both called as expected with the correct params. 

It sounds like it would be better to nest these chained functions calls instead?

Reply:
THis was resolved on Friday